shift-react-app/src/steps.ts

```diff
@@ -0,0 +1,65 @@
+// tslint:disable:no-console
+import {
+  readFileSync,
+  writeFileSync,
+} from 'fs';
+import { join } from 'path';
+import findLastIndex from 'lodash.findlastindex';
+import { spawn } from 'child_process';
+
+export function setupProxy() {
+  const proxyFile = join('src', 'setupProxy.js');
+  const proxyCode = `const { setupProxy } = require('@binaris/shift-local-proxy');
+module.exports = setupProxy(__dirname);
+`;
+  writeFileSync(proxyFile, proxyCode, { flag: 'wx' });
+  console.log(`Created ${proxyFile}, please commit this file`);
```

Comment:
I usually prefer not to log from functions in this level, they're easier to test if they return a value which could be logged by the "runner".